src/hypermail.c

```diff
@@ -365,9 +380,13 @@ int main(int argc, char **argv)
     }
 
 #ifdef HAVE_LOCALE_H
-	if (!setlocale(LC_ALL, locale_code)) {
+    if (!setlocale(LC_ALL, locale_code)) {
+	char *locale_code_utf8 = utf8locale(locale_code);
+	if (!setlocale(LC_ALL, locale_code_utf8)) {
 	    snprintf(errmsg, sizeof(errmsg), "WARNING: locale \"%s\", not supported.\n", locale_code);
 	    fprintf(stderr, "%s", errmsg);/* AUDIT biege: avoid format-bug warning */
```

Comment:
This patch look s ok. Could you explain what is "AUDIT biege"?

Reply:
No idea.  That's not code that I added or touched.